doc/sphinx/proofs/writing-proofs/proof-mode.rst

```diff
@@ -222,16 +222,12 @@ When the proof is completed, you can exit proof mode with commands such as
    This command is available in proof mode to give up
    the current proof and declare the initial goal as an axiom.
 
-.. cmd:: Abort {? {| All | @ident } }
+.. cmd:: Abort {? All }
 
    Cancels the current proof development, switching back to
    the previous proof development, or to the Coq toplevel if no other
    proof was being edited.
 
```

Comment:
```suggestion
   Cancels the current proof development, switching back to
   the previous proof development, or to the Coq toplevel if no other
   proof was being edited. See also :flag:`Nested Proofs Allowed`.

```

Reply:
Why though?

Reply:
With default options, `Abort` and `Abort All` have the same meaning no? So they only differ in behaviour when `Nested Proofs Allowed` is set. Therefore a user might read this and think "Why are there two different Abort commands?". The answer is because `Nested Proofs Allowed` is a thing.